src/config/config.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  // eslint-disable-next-line no-undef
+  host: __DEV__ ? "http://10.0.2.2:3001" : ""

```

Comment:
is there a way to parametrize application build, so I can pass the host url as a parameter?